DEVELOPMENT.md

```diff
@@ -3,6 +3,27 @@
 This doc explains the development workflow so you can get started
 [contributing](CONTRIBUTING.md) to Skaffold!
 
+
+## Requirements
+
+You must install these tools:
+
+1. [`go`](https://golang.org/doc/install): The language skaffold is
+   built in
+1. [`git`](https://help.github.com/articles/set-up-git/): For source control
+1. [`dep`](https://github.com/golang/dep): For managing external Go
+   dependencies. - Please Install dep v0.5.0 or greater.
+1. [`make`](https://www.gnu.org/software/make/): For building skaffold.
+1. [`golangci-lint`](https://github.com/golangci/golangci-lint): You can use the
+   helper [script](./hack/install_golint.sh) to get golangci-lint.
+
```

Comment:
remove new line? to move the 6 th point to the same block?

Reply:
the mdlint rule mentions empty line before and after "shell" block.

Reply:
will fix it.